tabs/onboard_logging.js

```diff
@@ -110,7 +110,7 @@ TABS.onboard_logging.initialize = function (callback) {
              * The best we can do on those targets is check the BLACKBOX feature bit to identify support for Blackbox instead.
              */
             if (BLACKBOX.supported || DATAFLASH.supported 
-                    || semver.gte(CONFIG.flightControllerVersion, "1.5.0") && semver.lte(CONFIG.flightControllerVersion, "1.10.0") && BF_CONFIG.features.isEnabled('BLACKBOX')) {
+                    || semver.gte(CONFIG.flightControllerVersion, "1.5.0") && semver.lte(CONFIG.flightControllerVersion, "1.10.0") && FEATURE_CONFIG.features.isEnabled('BLACKBOX')) {
```

Comment:
probably.  I'll be stripping a out all the old conditional code that's not currently getting in the way in later commits.